recipes/icc/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class ICCConan(ConanFile):
+    name = 'icc'
+    homepage = 'https://github.com/redradist/Inter-Component-Communication'
+    license = 'MIT'
+    url = 'https://github.com/conan-io/conan-center-index'
+    description = "I.C.C. - Inter Component Communication, This is a library created to simplify communication between " \
+                  "components inside of single application. It is thread safe and could be used for creating " \
+                  "components that works in different threads. "
+    topics = ("thread-safe", "active object")
+    settings = {"os": ["Windows", "Linux"], "compiler": None, "build_type": None, "arch": None}
```

Comment:
```suggestion
    settings = "os", "compiler", "build_type", "arch"
```

Raise an Invalid Configuration Instead

Reply:
Is there some suggestion for it in documentation ?

Reply:
Not directly documented, but is the pattern used for all recipes merged into CCI. 

The reason is simple, the Conan Center Bot skips your failed job when a ConanInvalidConfiguration is raised. However, when you limit your OS range by settings, Conan raises a different error, which is not ignored by the Bot and exits your build as failed.

Reply:
Done